src/interface/desktop/main.js

```diff
@@ -256,7 +256,7 @@ function pushDataToKhoj (regenerate = false) {
     .catch(error => {
         console.error(error);
         state["completed"] = false;
-        if (error?.response?.status === 429 && (BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('config')))) {
+        if (error?.response?.status === 429 && (BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('settings')))) {
             state["error"] = `Looks like you're out of space to sync your files. <a href="https://app.khoj.dev/settings#subscription">Upgrade your plan</a> to unlock more space.`;
             const win = BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('config'));
```

Comment:
This line may need to be updated too?

```suggestion
            const win = BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('settings'));
```

Reply:
sorry i forget that can fix now??

Reply:
That would be great!

Reply:
@debanjum i have added minor changes let me know